src/components/Votes.jsx

```diff
@@ -0,0 +1,7 @@
+const Votes = ({votes}) => {
+    return (
+        <p> Votes: {votes}</p>
+    )
+}
+
+export default Votes;
```

Comment:
Any particular reason this was put into it's own component instead of being with the rest of the article data?

Reply:
My plan is to handle add votes later, so I created this component.